server/chat/backend/agent/skills/integrations/victorops/SKILL.md

```
@@ -0,0 +1,41 @@
+  - get_victorops_teams
+index: "On-call incident management -- query recent incidents, team rosters, and on-call schedules from Splunk On-Call"
+rca_priority: 4
+allowed-tools: get_victorops_incidents, get_victorops_teams
+metadata:
+  author: aurora
+  version: "1.0"
+---
+
+# Splunk On-Call Integration
+
+## Overview
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)